Automattic/vip-go-mu-plugins
# Update/jetpack 10.1

## Description

Adding the release version of Jetpack 10.1 to the 10.1 branch.

## Changelog Description

### Plugin Updated: Jetpack 10.1

Added release version of Jetpack 10.1

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out 
2. Pin your testing site to the new subtree version: define( 'VIP_JETPACK_PINNED_VERSION', '10.1' );
3. Go to the Jetpack dashboard in wp-admin and ensure version 10.0-beta is active
4. Ensure everything else works as expected
